pom.xml

```diff
@@ -60,6 +60,16 @@
                 <type>pom</type>
                 <scope>import</scope>
             </dependency>
+            <dependency>
+                <groupId>com.github.workerframework.testing</groupId>
+                <artifactId>workerframework-testing-util</artifactId>
+                <version>1.0.0-SNAPSHOT</version>
+            </dependency>
+            <dependency>
+                <groupId>com.github.workerframework.testing</groupId>
+                <artifactId>workerframework-testing-integration</artifactId>
```

Comment:
Unlike normal dependencies, the `<dependencyManagement>` section should be in alphabetic order.